.github/workflows/repo2cpe-download.yaml

```diff
@@ -30,9 +30,16 @@ jobs:
           jq empty "$f"
         done
 
+    - name: Send Slack notification on failure
```

Comment:
Maybe this is what you mean? @jvdm  I've added both if failure() to download and upload steps

Reply:
Yes, but I am not sure you need multiple `if: failure()` steps; I thought only one at the end would suffice and run when previous actions fail.

Reply:
This is what would happen in my understanding, if the download fails: 

1. Send slack notification on failure
2. Upload is skipped
3. Send Slack notification on gsutil failure

I tried it out here: https://github.com/tommartensen/test-gh-actions/actions/runs/5442141681/jobs/9896975838.

Reply:
IMHO you should keep only one Slack message with `faillure()` at the end of the job. 
It might be possible do to some if logic magic with steps output to skip the second Slack message, but I don't see the benefit of investing too much time into that :)

Reply:
I have tested a new change, one failure() at the end is enough. As long as one of the steps failed, the Slack message will be posted. 
https://github.com/stackrox/stackrox/actions/runs/5447455127/jobs/9909470651